src/Unity.Mathematics/Geometry/MinMaxAABB.cs

```diff
@@ -1,49 +1,331 @@
 using System;
+using System.Runtime.CompilerServices;
 using static Unity.Mathematics.math;
 
 namespace Unity.Mathematics.Geometry
 {
+    /// <summary>
+    /// Axis aligned bounding box (AABB) stored in min/max form.
+    /// </summary>
+    /// <remarks>
+    /// Axis aligned bounding boxes (AABB) are boxes where each side is parallel with one of the Cartesian coordinate axes
+    /// X, Y, and Z. AABBs are useful for approximating the region an object (or collection of objects) occupy and quickly
+    /// testing whether or not that object (or collection of objects) is relevant. Because they are axis aligned, they
+    /// are very cheap to construct and perform overlap tests with them.
+    /// </remarks>
     [System.Serializable]
     internal struct MinMaxAABB : IEquatable<MinMaxAABB>
```

Comment:
The Physics AABB is public, this one is internal. If the plan is to remove the physics one, this one needs to be public as well.

Reply:
It will remain internal until we're ready to release.  I don't want to get caught with accidentally releasing something that wasn't ready and deal with trying to close pandora's box.